openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -178,6 +180,32 @@ const Carousel = {
                 }
             });
         }
+
+        function accessibilitySetup() {
+
+            function setA11yAttributes() {
+                //Ensuring offscreen elements do not receive focus
+                $(`${selector} [aria-hidden="true"] a[href]`).attr('tabindex', -1);
+                $(`${selector} [aria-hidden="false"] a[href]`).removeAttr('tabindex');
+
+                //Ensuring any disabled button elements do not receive focus
+                $(`${selector} button[aria-disabled="true"]`).attr('tabindex', -1);
+                $(`${selector} button[aria-disabled="false"]`).removeAttr('tabindex');
+            }
+
+            $(`${selector} [aria-live]`).removeAttr('aria-live');
+            $(`${selector} .slick-track`).attr('role','list').attr('aria-label',title);
+            setA11yAttributes();
+
+            $(`${selector} button.slick-prev`).on('click', function () {
+                $(`${selector} button.slick-next`).focus();
+            });
+            $(`${selector} button.slick-next`).on('click', function () {
+                $(`${selector} button.slick-prev`).focus();
+            });
+            $(selector).on('afterChange', setA11yAttributes);
```

Comment:
Please add an inline comment explaining the afterChange event, preferably pointing to some documentation.

Reply:
Wouldn't this bind multiple click events to the next and previous buttons?